spec/array_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'array'
+
+describe Array do
+
+  it 'uses the ruby inject method' do
+    expect([1,2,3,4].inject(0){ |result, element| result + element}).to eq(10)
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space missing inside }.